juspay/hyperswitch
# refactor(scheduler): join frequency and count in `RetryMapping`

## Type of Change



## Description

frequency and count are different field in RetryMapping that are conjoined to simplify the logic to set schedule time

## How did you test it?

- Run a scheduler
- Run a consumer
- Create a MCA(Cybersource)
- Create a Card Non-3DS payment
- Status should be succeeded
- Do a DB query to check whether process tracker is created for Payment Sync 
- PSync tracker must be present against the transaction made

Note: This affect scheduling of Process Trackers

Testcases added in the linked issue
## Checklist
